site/en/blog/permissions-chip/index.md

```diff
@@ -0,0 +1,142 @@
+in an accumulated way how Windows users reacted to the geolocation and the notification prompts:
+
+<div class="table-wrapper scrollbar">
+<table>
+<thead>
+<tr>
+<th>Action</th>
+<th>% of Geolocation Prompts</th>
+<th>% of Notification Prompts</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Allow</td>
+<td>8.09%</td>
+<td>6.69%</td>
+</tr>
+<tr>
+<td>Block</td>
+<td>6.47%</td>
+<td>9.20%</td>
+</tr>
+<tr>
+<td>Dismiss</td>
+<td>36.12%</td>
+<td>35.76%</td>
+</tr>
+<tr>
+<td>Ignore</td>
+<td>49.22%</td>
+<td>47.19%</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+Given this ignore/dismiss rate of ~85%, and especially given how much the prompt stands out and
+insists on users making a decision immediately, there is a conflict between the level of urgency
+assumed by the browser versus the user's preference for waiting to make a decision. This creates the
+perception that it's "annoying" for a site to ask for a permission.
+
+## New design
+
+From Chrome 97 on, we therefore introduced an animated chip UI which appears next to the lock
+whenever a permission is requested. This consists of an icon and label of the permission being
+requested. Our aim was to improve the experience of web browsing without being distracted by
+(generally unnecessary) permission requests for the vast majority of users who ignore or dismiss the
+current prompt.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/SsFtKKnZzbkAovNkX4Lt.png", alt="ALT_TEXT_HERE", width="653", height="229" %}
+
+The existing prompt bubble will be shown when the request chip is clicked (if not already shown) and
+the request UI is automatically augmented with the request bubble based on the heuristics listed
+below:
+
+- The permission was triggered through a user gesture when interacting with the site itself rather
+  than automatically triggered by the site.
+- The permission is deemed essential and generally non-spammy. Currently this includes camera,
+  microphone, and camera paired with microphone.
+
+### Flow of the new design
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/amp4hx5iei2FvfXFt8gK.png", alt="ALT_TEXT_HERE", width="800", height="445" %}
+
+For non-essential permissions not triggered by a gesture, the prompt no longer intrudes on the site
+contents, and is not insistent on an immediate decision. The user can ignore the request chip until
+they have enough information to make a decision.
+
+With no interaction, after a short delay, the request chip will automatically collapse to just a
+blocked icon (to indicate the permission being temporarily blocked), before being dismissed
+entirely. The aim is to get out of the way of users who choose not to make a decision and can do so
+without any interaction.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/0xzCGLqbOFMcDGVvM1Z2.png", alt="ALT_TEXT_HERE", width="722", height="180" %}
+
+### Expected short-term impact
+
+In the short-term, until users get used to the new UI, it is likely that site owners will observe
```

Comment:
```suggestion
In the short term, and until users get used to the new UI, it is likely that site owners will observe
```

Reply:
short-term vs. short term. 

This is one of the more confusing rules of English. When the phrase is in a noun position (in this case it's the subject of the preposition) it does not get a hyphen. If it is in an adjective position it does. For example, "It was a short-term gain."